### DISCUSSION TITLE
@alwatr/icon

### DISCUSSION THREAD

**Discussion by alimd** [AliMD/alwatr#109] [Ideas]
combine [ion-icon](https://ionic.io/ionicons) with lit-element and [css.gg](https://css.gg/) collection.

**Comment by MM25Zamanian** [comment]
[iconsax](https://iconsax.io/) ?

**Comment by alimd** [comment]
https://iconmonstr.com/

**Comment by alimd** [comment]
https://www.streamlinehq.com/icons

**Comment by alimd** [comment]
https://github.com/feathericons/feather/tree/master/icons

**Comment by alimd** [comment]
https://github.com/google/material-design-icons

**Comment by alimd** [comment]
عجب پروژه عجیبیه!!
دندونام

https://typogram.github.io/Anicons/
